sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md

```diff
@@ -4,11 +4,11 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added support for the `2023-11-01` service version.
+- Added a new method `ConfigurationClient.GetLabels` to support listing labels.
+- Added new classes `Label`, `LabelSelector`, and a new enum `LabelFields`.
+- Added a new property `TagsFilter` to `SettingSelector` type to support filtering settings or revisions with tags filter.
+- Added a new property `Tags` to `ConfigurationSettingsFilter` to support filtering settings with tags filter for a configuration snapshot.
```

Comment:
We generally want to scope this to the level that a developer using the library would care about.  and focus on scenarios:

  - Added support for listing labels
  - Added support for filtering by tags

(was tag filtering part of the GA service version?  I thought it was in preview)

Reply:
Thanks. I've gone ahead and updated the language to reflect this.